base/inference.jl

```diff
@@ -2415,6 +2416,7 @@ function inlineable(f::ANY, e::Expr, atype::ANY, sv::StaticVarInfo, enclosing_as
 
     body = Expr(:block)
     body.args = ast.args[3].args::Array{Any,1}
+    propagate_inbounds, _ = popmeta!(body, :propagate_inbounds)
```

Comment:
@JeffBezanson For some reason, unpacking returned Tuples like this doesn't seem to exist elsewhere in `inference.jl`. Is there a reason it is avoided?